tests/test_compositor.py

```diff
@@ -0,0 +1,40 @@
+from textual.app import App, ComposeResult
+from textual.containers import Container
+from textual.widgets import Static
+
+
+async def test_compositor_scroll_placements():
+    """Regression test for https://github.com/Textualize/textual/issues/5249"""
```

Comment:
```suggestion
    """Regression test for https://github.com/Textualize/textual/issues/5249
    The Static should remain visible.
    """
```

Reply:
The docstrings are visible in the snapshot report. If a snapshot fails, it's not always obvious whether what you're looking at is expected or not. A little comment like this can make it immediately obvious, and beats having to review and understand the code in the test.

Reply:
That's not a snapshot test, but I'm not apposed to the extra text.